org.lflang.tests/src/org/lflang/tests/LFParsingTest.java

```diff
@@ -26,6 +26,17 @@
     private ParseHelper<Model> parseHelper;
 
 
+    @Test
+    public void testLexingEmptyTargetProperties() throws Exception {
+        assertNoParsingErrorsIn("target C { };      \nreactor Foo {}");
+        assertNoParsingErrorsIn("target C {,};      \nreactor Foo {}");
```

Comment:
With my suggested change, this would yield something like:
```
lfc: error: extraneous input ',' expecting '}'
--> foo.lf:1:12
  |
1 | target C { , }
  |            ^ extraneous input ',' expecting '}'
  |
2 | 
```